creation/interfaces-and-networks.md

```diff
@@ -692,3 +681,164 @@ documentation](https://github.com/openshift/sriov-network-operator/blob/master/d
+metadata:
+  name: policy-1
+  namespace: sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  mtu: 9000
+  nicSelector:
+    pfNames:
+    - ens1f0
+  nodeSelector:
+    sriov: "true"
+  numVfs: 8
+  priority: 90
+  resourceName: sriov_net
+```
+
+The policy above will configure the `SR-IOV` device plugin, allowing the
+PF named `ens1f0` to be exposed in the SRIOV capable nodes as a resource named
+`sriov_net`.
+
+It will also render the `spec` for an associated `SriovNetworkNodeState` object,
+which looks like:
+
+```yaml
+apiVersion: v1
+items:
+- apiVersion: sriovnetwork.openshift.io/v1
+  kind: SriovNetworkNodeState
+  metadata:
+    name: sriov-worker
+    namespace: sriov-network-operator
+    ownerReferences:
+    - apiVersion: sriovnetwork.openshift.io/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: SriovNetworkNodePolicy
+      name: default
+  spec:
+    interfaces:
+    - mtu: 9000
+      name: ens1f0
+      numVfs: 8
+      pciAddress: 0000:65:00.0
+      vfGroups:
+      - deviceType: vfio-pci
+        resourceName: sriov_net
+        vfRange: 0-7
+  status:
+    interfaces:
+    - Vfs:
+      - deviceID: "1016"
+        driver: vfio-pci
+        pciAddress: 0000:65:00.2
+        vendor: 15b3
+        vfID: 0
+        deviceID: "1015"
+      ...
+  syncStatus: Succeeded
+```
+
+#### Start an SR-IOV VM
+
+Once all the SR-IOV components are deployed, it is needed to indicate how to
+configure the SR-IOV network. Refer to the following
+`SriovNetwork` for an example:
+
+```yaml
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  annotations:
+    operator.sriovnetwork.openshift.io/last-network-namespace: default
+  finalizers:
+  - netattdef.finalizers.sriovnetwork.openshift.io
+  generation: 2
+  name: sriov-mlx
+  namespace: sriov-network-operator
+spec:
+  ipam: |
+    {}
+  networkNamespace: default
+  resourceName: sriov_net ??
+  spoofChk: "off"
+
+```
+
+Finally, to create a VM that will attach to the aforementioned Network, refer
+to the following VMI spec:
+
+```yaml
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-perf
+  name: vmi-perf
+spec:
+  domain:
+    cpu:
+      sockets: 2
+      cores: 1
+      threads: 1
+      dedicatedCpuPlacement: true
+    resources:
+      requests:
+        memory: "4Gi"
+      limits:
+        memory: "4Gi"
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {}
+        name: default
+      - name: sriov-net
+        sriov: {}
+      rng: {}
+    machine:
+      type: ""
+  networks:
+  - name: default
+    pod: {}
+  - multus:
+      networkName: default/sriov-mlx
+    name: sriov-net
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - containerDisk:
+      image: quay.io/kubevirt/fedora-cloud-container-disk-demo:devel
```

Comment:
I don't think we have this on Quay. And `devel` is only on local builds.

`registry.hub.docker.com/kubevirt/fedora-cloud-container-disk-demo:latest`

https://hub.docker.com/layers/kubevirt/fedora-cloud-container-disk-demo/latest/images/sha256-d9ec8640375d27000e32c6491f3b90ee0c433f07e2f0f8b6faae94ed041dee2f?context=explore